content/en/blog/2022-08-26-manage-kyverno-policies-as-ocirepositories/index.md

```diff
@@ -0,0 +1,207 @@
+
+{{< imgproc twitter-ss Resize 400x >}}
+{{< /imgproc >}}
+
+{{< imgproc meme Resize 400x >}}
+{{< /imgproc >}}
+
+Today, we’ll leverage the OCI Repositories feature to apply Kyverno policies stored in an OCI registry into the Kubernetes cluster.
+
+First, we need to install Flux CLI, please see the [installation](https://fluxcd.io/docs/installation/) page to find out details.
+
+Next, we should have Kubernetes cluster running, we’ll be using [KinD](https://kind.sigs.k8s.io/docs/user/quick-start#configuring-your-kind-cluster) for this purpose.
+
+```shell
+$ kind create cluster
+```
+
+Once the cluster has been provisioned successfully, we need to install Flux components into it by simply running the command below:
+
+```shell
+$ flux bootstrap github \
+  --owner=developer-guy \
+  --repository=flux-kyverno-policies \
+  --path=clusters/local \
+  --personal
+```
+
+> ⚠️ **Note:** Don't forget to change the values with your own details!
+
+This command will install Flux and create necessary files for us and push them into the repository.
+
+Next, we should install Kyverno first. As we do everything GitOps in mind, we’ll also install Kyverno with Flux. In order to do that, we use HelmRepository and HelmRelease resources:
+
+```yaml
+---
+apiVersion: source.toolkit.fluxcd.io/v1beta2
+kind: HelmRepository
+metadata:
+  name: kyverno
+  namespace: flux-system
+spec:
+  interval: 10m
+  url: https://kyverno.github.io/kyverno/
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: kyverno
+  namespace: flux-system
+spec:
+  interval: 10m
+  targetNamespace: kyverno
+  install:
+    createNamespace: true
+  chart:
+    spec:
+      chart: kyverno
+      sourceRef:
+        kind: HelmRepository
+        name: kyverno
+```
+
+Do not forget to check whether everything works fine before moving into the next steps:
+
+```shell
+$ flux get helmreleases kyverno
+NAME    REVISION        SUSPENDED       READY   MESSAGE
+kyverno v2.5.3          False           True    Release reconciliation succeeded
+```
+
+Now, we are ready to create an OCI image to store my Kyverno policies.
+
+> ⚠️  You can find all the code examples in [GitHub](https://github.com/developer-guy/flux-kyverno-policies).
+
+In order to do that, we will clone our repository that holds Kyverno policies and create an OCI image to store them
+
+```shell
+$ git clone https://github.com/developer-guy/my-kyverno-policies.git
+$ cd my-kyverno-policies
+```
+
+> ⚠️ We are expecting that some other team “DevSecOps” is responsible to maintain and publish the policies to GHCR
+
+Next, we will be applying cert-manager policies into the cluster as an example:
```

Comment:
I'm confused by this step. The shell command you're showing beneath this line is a follow-on from your previous step where you cloned your developer-guy/my-kyverno-policies repository and moved into that directory. That repository has one Kyverno policy. There's nothing which appears to have anything to do with cert-manager here.